backend/src/main/java/de/neuefische/backend/controller/PlanController.java

```diff
@@ -0,0 +1,36 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.model.UpdatePlan;
+import de.neuefische.backend.model.Plan;
+import de.neuefische.backend.service.PlanService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/plan")
+@RequiredArgsConstructor
+public class PlanController {
+    private final PlanService planService;
+
+    @GetMapping
+    public List<Plan> getAllPlans() {
+        return planService.findAllPlans();
+    }
+
+    @PostMapping
+    public Plan postPlan(@RequestBody Plan plan) {
+        return planService.addPlan(plan);
+    }
+
+    @DeleteMapping("{id}")
+    public void deletePlan(@PathVariable String id) {
+        planService.deletePlan(id);
+    }
+
+    @PutMapping("{id}")
```

Comment:
hier das gleiche. Es funktioniert denke ich jetzt zwar, aber ich bin in der Vergangenheit trotzdem dann später in Probleme gekommen, weil der slash gefehlt hat. Schreibe lieber 
`@PutMapping("/{id}")`